sdk/storage/azure-storage-file-datalake/README.md

```diff
@@ -375,6 +325,17 @@ When interacting with data lake using this Java client library, errors returned
 status codes returned for [REST API][error_codes] requests. For example, if you try to retrieve a file system or path that
 doesn't exist in your Storage Account, a `404` error is returned, indicating `Not Found`.
 
+### Default HTTP Client
```

Comment:
Looks like this has italicized text starting from line 123 `- The _filesystem used via 'DataLakeFileSystemClient'`. There's an underscore before filesystem that's causing this.

Reply:
updated